lib/solaar/configuration.py

```diff
@@ -23,9 +23,12 @@
 from logging import DEBUG as _DEBUG
 from logging import INFO as _INFO
 from logging import getLogger
+from threading import Lock as _Lock
+from threading import Timer as _Timer
 
 import yaml as _yaml
 
+from gi.repository import GLib
```

Comment:
This adds a dependency on PyGObject (which brings dozens of Gtk packages) even for CLI. According to the [installation.md](https://github.com/pwr-Solaar/Solaar/blob/master/docs/installation.md), GObject is needed only for GUI, and it would be better to keep it like this.

Reply:
Seems reasonable.  Moving the import to the place where it is used should only pull in Glib stuff when a deferred write is needed should fix the issue.

Reply:
I’m not sure, if I understand the code correctly, deferred write may be used even with CLI…?

Reply:
https://github.com/pwr-Solaar/Solaar/pull/1816